crates/red_knot_python_semantic/resources/mdtest/type_properties/is_subtype_of.md

```diff
@@ -1,60 +1,74 @@
 # Subtype relation
 
-## Basic types
+The `is_subtype_of(S, T)` relation below checks if type `S` is a subtype of type `T`.
 
-```py
-from typing_extensions import Literal
-from knot_extensions import is_subtype_of, static_assert
-
-static_assert(is_subtype_of(str, object))
-static_assert(is_subtype_of(int, object))
-static_assert(is_subtype_of(bool, object))
+A fully static type `T` is a subtype of another fully static type `S` iff the set of values
+represented by `S` is a subset of the set of values represented by `T`.
 
-static_assert(is_subtype_of(bool, int))
-static_assert(is_subtype_of(TypeError, Exception))
-static_assert(is_subtype_of(FloatingPointError, Exception))
+See the [typing documentation] for more information.
 
-static_assert(not is_subtype_of(object, int))
-static_assert(not is_subtype_of(int, str))
-static_assert(not is_subtype_of(int, Literal[1]))
+## Basic builtin types
 
-static_assert(not is_subtype_of(Literal[1], str))
-static_assert(not is_subtype_of(Literal[1, 2], Literal[1]))
-static_assert(not is_subtype_of(Literal[1, 2], Literal[1, 3]))
-```
+```py
+from knot_extensions import is_subtype_of, static_assert
 
-## Non-fully-static types
+static_assert(is_subtype_of(bool, bool))
+static_assert(is_subtype_of(bool, int))
+static_assert(is_subtype_of(bool, object))
 
-`Any`, `Unknown`, `Todo` and derivatives thereof do not participate in subtyping.
+static_assert(is_subtype_of(int, int))
+static_assert(is_subtype_of(int, object))
 
-```py
-from typing import Any
-from typing_extensions import Literal
-from knot_extensions import Unknown, is_subtype_of, static_assert
+static_assert(is_subtype_of(object, object))
 
-static_assert(not is_subtype_of(Unknown, Unknown))
-static_assert(not is_subtype_of(Unknown, Literal[1]))
-static_assert(not is_subtype_of(Literal[1], Unknown))
+static_assert(not is_subtype_of(int, bool))
+static_assert(not is_subtype_of(int, str))
+static_assert(not is_subtype_of(object, int))
 
-static_assert(not is_subtype_of(Any, Any))
-static_assert(not is_subtype_of(Any, Literal[1]))
+# See: [special case for float and complex]
```

Comment:
This might answer my question above.  Is this meant to be a link?  There is no link footnote giving it a URL.  And since this is in a code block it won't be rendered as a link even if there were.  Maybe replace this with (or add) a URL?

Reply:
Oh. This was supposed to link to https://typing.readthedocs.io/en/latest/spec/special-types.html#special-cases-for-float-and-complex, but since it only appears in a code block, mdformat removed my link at the bottom of the document :angry:

Reply:
Fixed, thanks.